arbor/morph/region.cpp

```diff
@@ -132,6 +132,43 @@ mcable_list remove_cover(mcable_list cables, const morphology& m) {
     return merge(cables);
 }
 
+mcable_list remove_redundancy(mcable_list cables, const morphology& m) {
```

Comment:
Would `remove_covered_points` be a better name for this function?

How about this for a comment above the function?
```C++
// Remove zero-length regions from a sorted cable_list that are in the cover of another region in the list.
```

Reply:
Okay, I'll modify it